pkg/metrics/metrics.go

```diff
@@ -16,11 +16,18 @@ const (
 	HCOMetricOverwrittenModifications = "overwrittenModifications"
 	HCOMetricUnsafeModifications      = "unsafeModifications"
 	HCOMetricHyperConvergedExists     = "HyperConvergedCRExists"
+	HCOMetricSystemHealthStatus       = "systemHealthStatus"
 
 	HyperConvergedExists    = float64(1)
 	HyperConvergedNotExists = float64(0)
 )
 
+const (
+	SystemHealthStatusHealthy float64 = iota
```

Comment:
Why are we setting this to `float64` type and not `int`?

Reply:
Because Prometheus values are `float64`